### PR TITLE
update parkour

### PR DESCRIPTION
**Pull request by HentaiStorm**
## About The Pull Request

Perk 'Parkour' gives the ability to walk around low walls and railings

## Why It's Good For The Game

1) Climb up previously inaccessible ledges
2) Escape from roaches

## Changelog
:cl:
add: Added perk 'parkour' ability to walk around low walls and railings
/:cl:

### CODE REVIEW

File: code/modules/multiz/movement.dm [hunk 1]
```diff
@@ -222,6 +222,12 @@
 /mob/living/carbon/human/can_fall(turf/below, turf/simulated/open/dest = src.loc)
 	// Special condition for jetpack mounted folk!
 	if (!restrained())
+		if(stats.getPerk(PERK_PARKOUR))
+			for(var/obj/structure/low_wall/LW in view(src, 1))
```
Comment (MLGTASTICa) [inline code/modules/multiz/movement.dm:226]: why for and not if ? seems like unnecesarry looping.

**Comment by Alliostra** [comment]
this is very cool, maybe the same should be possible with magboots? And maybe you should be slowed down while climbing so you can't just run off a ledge at full speed and run to the other side.

**Comment by Omni44** [comment]
Will that also make possible to climb onto low walls and railings from below while standing on the table? Right now railings prevent you from climbing completely if I recall correctly, that'd be a nice niche.

File: code/datums/perks/oddity.dm [hunk 2]
```diff
@@ -63,7 +63,7 @@
 
 /datum/perk/oddity/parkour
 	name = "Parkour"
-	desc = "You can climb tables and ladders faster."
+	desc = "You can climb tables and ladders faster. Also clings to railings and low walls."
```
Comment (TheShown911) [inline code/datums/perks/oddity.dm:66]:
```suggestion
	desc = "You can climb tables and ladders faster and also cling to railings and low walls."
```

File: code/modules/multiz/movement.dm [hunk 3]
```diff
@@ -220,10 +220,20 @@
 
 
 /mob/living/carbon/human/can_fall(turf/below, turf/simulated/open/dest = src.loc)
-	// Special condition for jetpack mounted folk!
-	if (!restrained())
-		if (CanAvoidGravity())
```
Comment (TheShown911) [inline code/modules/multiz/movement.dm:225]: You would need to put back this `CanAvoidGravity()` check somewhere in here (if it's even used).

**Review by TheShown911 — APPROVED** [review]
Very cool, slowdown may be needed to balance it out but that can be done later.

**Pull request merged**